build.js

```diff
@@ -143,7 +142,7 @@ function buildlocale (source, locale) {
     .use(layouts({
       engine: 'handlebars',
       pattern: '**/*.html',
-      partials: mapHandlebarsPartials(metalsmith, 'layouts', 'partials'),
+      partials: 'layouts/partials',
```

Comment:
Does `metalsmith-layouts` handle this in a windows compat manner or should you use `path.join()`?